docs/Toy/toy/tests/test_toy.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from ..parser import Parser
+
+
+def test_parse_ast():
+    ast_toy = Path() / 'docs' / 'Toy' / 'examples' / 'ast.toy'
+
```

Comment:
Can;t we do:
```
dir_to_scan = "......xxx.x.x.x.x.x.x..examples/ast_toy"
ast_toy = Path(dir_to_scan)
```
as in: https://pbpython.com/pathlib-intro.html
```
from pathlib import Path

dir_to_scan = "/media/chris/KINGSTON/data_analysis"
p = Path(dir_to_scan)
```

Reply:
I guess we could if you prefer it